deploy/configs/cognito_urls_config.py

```diff
@@ -44,73 +49,68 @@ def setup_cognito(
             'Value'
         ]
 
-    print(f'UI: {signin_singout_link}')
-    print(f'USERGUIDE: {user_guide_link}')
+    log.info(f'UI: {signin_singout_link}')
+    log.info(f'USERGUIDE: {user_guide_link}')
 
     cognito = boto3.client('cognito-idp', region_name=region)
-    try:
-        user_pool = cognito.describe_user_pool_client(UserPoolId=user_pool_id, ClientId=app_client)
+    user_pool = cognito.describe_user_pool_client(UserPoolId=user_pool_id, ClientId=app_client)
```

Comment:
You removed the general try/except because if it fails then the stack will rollback right?

Reply:
Nope, I removed it because it was only catching the exception printing and throwing it again which isn't required as the exception will currently bubble up and print the exact same information...

```
 except ClientError as e:
        print(f'Failed to setup cognito due to: {e}')
        raise e
```